perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -249,6 +302,10 @@ class GceVirtualMachine(virtual_machine.BaseVirtualMachine):
   BOOT_DISK_SIZE_GB = 10
   BOOT_DISK_TYPE = gce_disk.PD_STANDARD
 
+  _host_lock = threading.Lock()
+  deleted_hosts = set()
+  host_map = collections.defaultdict(list)
```

Comment:
Sorry if I am missing something obvious, but where is this 'list' initialized ?
From docs, The first argument provides the initial value for the default_factory attribute; it defaults to None.